frontend/packages/shared/src/queryInvalidator/SyncSuccessQueriesInvalidator.ts

```diff
@@ -30,7 +30,7 @@ export class SyncSuccessQueriesInvalidator extends Queue {
 
   // Maps folder names to their cache keys for invalidation upon sync success - can be extended to include more folders
   private readonly folderNameCacheKeyMap: Record<string, Array<QueryKey | string>> = {
-    layouts: [QueryKey.FormLayouts, '[org]', '[app]', '[layoutSetName]'],
+    layouts: [QueryKey.FormLayouts, '[org]', '[app]'],
```

Comment:
Removed `[layoutSetName]` as this is just mapped to the currently selected layout set, which, for this PR, causes the query not to get properly invalidated.

Reply:
One issue I see here is that this will invalidate all layouts rather than just the impacted ones. A possible solution could be having the backend return the impacted layoutsets, but I can imagine that this requires a larger refactoring 🤔 We could consider adding a TODO to remember to refactor this later to support passing a list of layoutsets. 🤷‍♂️